modules/api/src/main/java/net/countercraft/movecraft/MovecraftRepair.java

```diff
@@ -158,166 +158,187 @@ public Clipboard loadCraftRepairStateClipboard(Craft craft, Sign sign, String re
-                                itemToConsume = 330;
-                                qtyToConsume = 0.5;
-                            }
-                            if (itemToConsume == 193) {
-                                itemToConsume = 427;
-                                qtyToConsume = 0.5;
-                            }
-                            if (itemToConsume == 194) {
-                                itemToConsume = 428;
-                                qtyToConsume = 0.5;
-                            }
-                            if (itemToConsume == 195) {
-                                itemToConsume = 429;
-                                qtyToConsume = 0.5;
-                            }
-                            if (itemToConsume == 196) {
-                                itemToConsume = 430;
-                                qtyToConsume = 0.5;
-                            }
-                            if (itemToConsume == 197) {
-                                itemToConsume = 431;
-                                qtyToConsume = 0.5;
-                            }
-                            if (itemToConsume == 23) {
-                                Tag t = block.getNbtData().getValue().get("Items");
-                                ListTag lt = null;
-                                if (t instanceof ListTag) {
-                                    lt = (ListTag) t;
-                                }
-                                int numTNT = 0;
-                                int numFireCharges = 0;
-                                int numWaterBuckets = 0;
-                                if (lt != null) {
-                                    for (Tag entryTag : lt.getValue()) {
-                                        if (entryTag instanceof CompoundTag) {
-                                            CompoundTag cTag = (CompoundTag) entryTag;
-                                            if (cTag.toString().contains("minecraft:tnt")) {
-                                                numTNT += cTag.getByte("Count");
-                                            }
-                                            if (cTag.toString().contains("minecraft:fire_charge")) {
-                                                numFireCharges += cTag.getByte("Count");
-                                            }
-                                            if (cTag.toString().contains("minecraft:water_bucket")) {
-                                                numWaterBuckets += cTag.getByte("Count");
+                            switch (itemToConsume) {
+                                case 63:// signs
+                                    itemToConsume = 323;
+                                    break;
+                                case 68:
+                                    itemToConsume = 323;
+                                    break;
+                                case 93:// repeaters
```

Comment:
in the case that 2 keys map to the same operation, you can simply omit the `break;` statement and allow the execution to fall through